README.md

```diff
@@ -34,7 +34,7 @@ This library depends on the Fetch and Promise APIs. These will be expected to be
 
 ### Dependencies
 
-- Yarn
+- npm
```

Comment:
We can specify that it's npm 6, as npm 7 introduces a new lockfile format we don't want yet until everything works with node 16 (which includes npm 7 by default).

Reply:
Updated